ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -84,15 +84,15 @@ public static bool LocationsAreEqual(Location location1, Location location2)
                 for (var x = 0; x < gridWidth; x++)
                 {
                     var currentCell = row[x];
-                    cells.Add(new Cell(GetSourceDataCellState(currentCell), x, y));
+                    cells.Add(new Cell(GetCellState(currentCell), x, y));

```

Comment:
Try keep levels of abstraction consistent for easier reading

```suggestion
                    var cellState = GetCellState(currentCell);
                    cells.Add(new Cell(cellState, x, y));
```